### PR TITLE
Don't start health checks for staged tasks

### LINKED ISSUE #934: Staged Docker containers sometimes show red health icon
During staging the health checks should be ignored. But sometimes the UI shows them with a red health icon during staging.

### PR DESCRIPTION
**Pull request by sttts**
Fixes #934

### CODE REVIEW

**Comment by drexin** [comment]
@sttts Could you please add a test for this?

**Comment by sttts** [comment]
Sure.

**Comment by sttts** [comment]
Testing this is harder than expected. The HealthCheckActor does not allow to query whether there is a health check worker running for a specific task id.

Testing the opposite would be easy with akka testkit message observation, probably using expectMsgAllClassOf, but the non-existence of such a message is impossible to observe without a timeout (not nice).

Either we extend the actor in a way to be observable easier in tests or we just leave it with this oneliner patch.

**Comment by drexin** [comment]
Ok, I will make some changes to the actor and add tests. Once I have the the tests ready, I will merge this PR.

File: src/main/scala/mesosphere/marathon/health/HealthCheckActor.scala [hunk 1]
```diff
@@ -81,7 +81,7 @@ class HealthCheckActor(
   protected[this] def dispatchJobs(): Unit = {
     log.debug("Dispatching health check jobs to workers")
     taskTracker.get(appId).foreach { task =>
-      if (task.getVersion() == appVersion) {
+      if (task.getVersion() == appVersion && task.hasStartedAt) {
```
Comment (ConnorDoyle) [inline src/main/scala/mesosphere/marathon/health/HealthCheckActor.scala]: :+1:

**Pull request merged**